pkg/apis/aquasecurity/v1alpha1/common_types.go

```diff
@@ -1,8 +1,14 @@
 package v1alpha1
 
+import "time"
+
 // Scanner is the spec for a scanner generating a security assessment report.
 type Scanner struct {
 	Name    string `json:"name"`
 	Vendor  string `json:"vendor"`
 	Version string `json:"version"`
 }
+
+type GeneratedAt struct {
```

Comment:
Hey @danielsagi This is a tricky one. I should have put more details into the issue description:

Each Kubernetes object, including custom object such as VulnerabilityReport and ConfigAuditReport, has `CreationTimestamp` ObjectMeta property that can be used instead of defining our own. We also use the CreationTimestamp property in Octant plugin.

We actually created a similar thing to custom GeneratedAt property long time back to make sure this date is independent of the etcd system date, but it has its own drawbacks. For example, if you want to calculate content hash of a VulnerabilityReport to optimize storage, the GeneratedAt property would change the hash.

Reply:
Yes I know CreationTimestamp is in ObjectMeta, but there is no LastUpdatedTimestamp.
For example if we run multiple scans on the same deployment, the CreationTimestamp stays the same. as the resource was created firstly, and then updated.